libs/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+add_subdirectory("api")
+add_subdirectory("api_common")
+add_subdirectory("execution_context")
+add_subdirectory("platform")
+add_subdirectory("ebpfnetsh")
```

Comment:
does the order here matter?  Or can these be listed in alphabetical order?

Reply:
I think it is best practice to list dependencies first, but order usually does not matter except in (unusual) specific cases (for example: attempting to [query properties from a target](https://cmake.org/cmake/help/latest/command/get_target_property.html) that has not yet been defined).